src/k8s/pkg/k8sd/database/sql/migrations/worker-tokens/001-add-expiry.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE worker_tokens
+ADD COLUMN expiry DATETIME DEFAULT '2100-01-01 23:59:59';
```

Comment:
@bschimke95 what if the table `worker_tokens` has a column named `expiry` already.  Won't this fail?